crates/holochain/src/conductor/tests/network_info.rs

```diff
@@ -54,7 +54,7 @@ async fn network_info() {
 
     wait_for_integration(
         &conductors[1].get_dht_db(dna.dna_hash()).unwrap(),
-        100,
+        28,
```

Comment:
This was failing on CI and locally 100% of the time with the changes on this PR. It was effectively just ending up being a 10 second wait then quietly moving on which was kind of okay but it's better fixed